### PR TITLE
feat(hss): add web tamper protection resource support

### PR DESCRIPTION
**Pull request by jinyangyang222**
**What this PR does / why we need it**:
add web tamper protection resource support
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
add web tamper protection resource support
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
$ export HW_ENTERPRISE_PROJECT_ID_TEST=xxxxxxxxxx
$ export export HW_HSS_HOST_PROTECTION_HOST_ID=xxxxxxx
$ export HW_HSS_HOST_PROTECTION_QUOTA_ID=xxxxx
$ make testacc TEST="./huaweicloud/services/acceptance/hss" TESTARGS="-run TestAccWebTamperProtection_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/hss -v -run TestAccWebTamperProtection_basic -timeout 360m -parallel 4
=== RUN   TestAccWebTamperProtection_basic
=== PAUSE TestAccWebTamperProtection_basic
=== CONT  TestAccWebTamperProtection_basic
--- PASS: TestAccWebTamperProtection_basic (25.48s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/hss       25.518s

```

### CODE REVIEW

File: huaweicloud/services/hss/resource_huaweicloud_hss_webtamper_protection.go [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+		EnterpriseProjectId: &epsId,
+		Body:                &requestBody,
+	}
+
+	_, err := client.SetWtpProtectionStatusInfo(&opts)
+	if err != nil {
+		return err
+	}
+
+	return nil
```
Comment (Lance52259) [inline huaweicloud/services/hss/resource_huaweicloud_hss_webtamper_protection.go]:
```suggestion
	return err
```
If the error is not formatted, returned directly just ok.
other places ditto

Comment (jinyangyang222) [reply huaweicloud/services/hss/resource_huaweicloud_hss_webtamper_protection.go]: done.

**Pull request merged**